resources/examples/simple-standalone.sh

```diff
@@ -1,80 +1,127 @@
 #!/bin/bash
-
+#foo
 # ARG_POSITIONAL_SINGLE([filename])
 # ARG_OPTIONAL_SINGLE([unit],[u],[What unit we accept (b for bytes, k for kilobytes, M for megabytes)],[b])
-# ARG_VERSION([echo $0 v0.1])
+# ARG_VERSION([echo "$0 v0.1"])
 # ARG_OPTIONAL_BOOLEAN([verbose])
-# ARG_HELP()
+# ARG_HELP([])
 # ARGBASH_SET_INDENT([  ])
 # ARGBASH_GO()
 # needed because of Argbash --> m4_ignore([
-### START OF CODE GENERATED BY Argbash v2.0.0 one line above ###
+### START OF CODE GENERATED BY Argbash v2.10.0 one line above ###
```

Comment:
Note for self: For some reasons, this doesn't get regenerated as a part of the release process.

Reply:
Yes, I believe both this and simple-parsing are missing from line 58 of resources/Makefile.  I'll put that in.